doc/cli.md

```diff
@@ -0,0 +1,230 @@
+# Command Line Interface
+
+Persephone supports the `persephone` CLI command with two subcommands: `validate` and `match`, which performs Statement Template validation and Pattern matching, respectively.
+
+To use the CLI, first run `make bundle`, then `cd` to `target/bundle`. You will then be able to run `/bin/persephone.sh`, which will accept a subcommand followed by arguments for Profiles, Statements, and so on. You can also provide the command with a `--help` or `-h` flag and it will list its subcommands.
+
+Note that Profile and Statement arguments accept URIs, e.g. filepaths, not the JSON data itself. In addition, some arguments (e.g. `--profile`) may be repeated to allow the CLI to accept multiple arguments.
+
+The `validate` subcommand validates a single Statement against Statement Templates in one or more Profiles. The following table shows its arguments:
+
+| Subcommand&nbsp;Argument | Description
+| :--                          | :--
+| `-p, --profile URI`          | Profile URI filepath/location; must specify one or more.
```

Comment:
If I specify multiple profiles does it validate/match against all or any?

Reply:
Yes all the profiles get smooshed into a single profile cosmos. Hence why the `-i` flag becomes particularly important in that case. It becomes necessary for Templates and Patterns that reference entities in other profiles.